src/map.cpp

```diff
@@ -156,6 +156,35 @@ void VoxelObject::MarkDirty()
 	MarkVoxelDirty(this->vox_pos);
 }
 
+/**
+ * Load a voxel object from the save game.
+ * @param ldr Input stream to read.
+ */
+void VoxelObject::Load(Loader &ldr)
+{
+	uint32 x = ldr.GetLong();
+	uint32 y = ldr.GetLong();
+	uint32 z = ldr.GetLong();
+
+	this->vox_pos.x = x >> 16;
+	this->vox_pos.y = y >> 16;
+	this->vox_pos.z = z >> 16;
+	this->pix_pos.x = x & 0xff;
+	this->pix_pos.y = y & 0xff;
+	this->pix_pos.z = z & 0xff;
```

Comment:
storing 8 bit values in 16 bits? nah :)
It would be in coaster cars I suspect, ie the other form of voxel objects.